kmip/services/kmip_client.py

```diff
@@ -859,3 +860,6 @@ def _set_variables(self, host, port, keyfile, certfile,
 
         self.password = conf.get_valid_value(
             password, self.config, 'password', conf.DEFAULT_PASSWORD)
+
+        self.timeout = conf.get_valid_value(
```

Comment:
What are the checks to verify that a valid value is configured? What if a value is less than or equal to zero?